recipe/meta.yaml

```diff
@@ -18,37 +17,22 @@ build:
     - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
   script:
     - {{ PYTHON }} -m pip install tools/pythonpkg -vv
-  ignore_run_exports:
-    - arrow-cpp
 
 requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
-    - pybind11                               # [build_platform != target_platform]
-    - arrow-cpp                              # [build_platform != target_platform]
+    - pybind11 >=2.6                         # [build_platform != target_platform]
     - setuptools_scm <7
-    - cmake
-    - ninja
-    - make
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
   host:
-    - pybind11 >=2.4
+    - pybind11 >=2.6
     - python
     - pip
     - setuptools_scm <7
```

Comment:
Why this pin?

Reply:
🤷 Ask upstream, this still exists in main: https://github.com/duckdb/duckdb/blob/080119274abade20de376777c2e26d0936d93ff9/tools/pythonpkg/setup.py#L351